mobile/src/main/res/drawable/list_arrow_activated_tinted.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<nine-patch xmlns:android="http://schemas.android.com/apk/res/android"
+    android:src="@drawable/list_arrow_activated"
+    android:tint="?attr/colorControlHighlight"
```

Comment:
Maybe could also use `?attr/colorControlActivated` here? Thoughts?

Reply:
I would go for colorControlHighlight